pkg/util/conntrack/conntrack_test.go

```diff
@@ -200,30 +208,52 @@ func TestDeleteUDPConnections(t *testing.T) {
 		name   string
 		origin string
 		dest   string
+		proto  v1.Protocol
 	}{
 		{
 			name:   "IPv4 success",
 			origin: "1.2.3.4",
 			dest:   "10.20.30.40",
+			proto:  v1.ProtocolUDP,
 		},
 		{
 			name:   "IPv4 simulated failure",
 			origin: "2.3.4.5",
 			dest:   "20.30.40.50",
+			proto:  v1.ProtocolUDP,
 		},
 		{
 			name:   "IPv6 success",
 			origin: "fd00::600d:f00d",
 			dest:   "2001:db8::5",
+			proto:  v1.ProtocolUDP,
+		},
+		{
+			name:   "IPv4 success",
```

Comment:
I guess we should update the name of the tests with the protocol to differentiate them